tools/addrundata/rundata.go

```diff
@@ -15,51 +15,52 @@ type parsedData struct {
 	testPlanID      string
 	testDescription string
 	testUUID        string
-	hasData         bool
 }
 
 // markdownRE matches the heading line: `# XX-1.1: Foo Functional Test`
 var markdownRE = regexp.MustCompile(`#(.*?):(.*)`)
 
 // fromMarkdown reads parsedData from README.md
-func (pd *parsedData) fromMarkdown(r io.Reader) error {
+func fromMarkdown(r io.Reader) (*parsedData, error) {
```

Comment:
nit: since this is now a top-level function, let's rename the functions to better disambiguate them

* fromMarkdown -> parseMarkdown
* fromCode -> parseCode

Reply:
Good call, done.